camera_system/color_changer.py

```diff
@@ -0,0 +1,65 @@
+"""画像変換モジュール.
+
+カメラから取得した画像を6色画像に変換する。
+@author kodama0720
+
+"""
+
+import cv2 as cv
+import numpy as np
+
+
+class ColorChanger:
+    """画像の色変換クラス.
+
+    Attributes:
+        __BLACK(ndarray): RGB値(黒)
+        __BGR_COLOR(ndarray): RGB値(赤、黄色、緑、青、白)
+        __LOWER(ndarray): HSV閾値下限(赤1、赤2、黄色、緑、青、白)
+        __UPPER(ndarray): HSV閾値上限(赤1、赤2、黄色、緑、青、白)
+
+    """
+
+    __BLACK = np.array([0, 0, 0])
+    __BGR_COLOR = np.array([[0, 0, 255], [0, 255, 255], [0, 255, 0],
+                           [255, 0, 0], [255, 255, 255]])
+    __LOWER = np.array([[0, 90, 0], [151, 90, 0], [16, 130, 0],
+                        [41, 70, 0], [104, 100, 0], [0, 0, 128]])
+    __UPPER = np.array([[15, 255, 255], [179, 255, 255], [40, 255, 255],
+                        [103, 255, 255], [150, 255, 255], [179, 70, 255]])
+
+    def change_color(self, path: str = "course.png") -> None:
+        """画像を6色画像に変換する関数.
+
+        Args:
+            path (str, optional): 入力画像ファイルのパス. Defaults to "course.png".
+
+        """
+        frame_mask = []
+        # 画像データの読み込み
+        img = cv.imread(path)
+        # BGR色空間からHSV色空間への変換
+        hsv = cv.cvtColor(img, cv.COLOR_BGR2HSV)
+        # 元画像と同じサイズの黒色画像を作成
+        result = np.zeros(
+            (img.shape[0], img.shape[1], img.shape[2]), np.uint8)
```

Comment:
```suggestion
        result = np.zeros(img.shape, np.uint8)
```

で問題ないならこっちの方がいいと思います。